EIPS/eip-2794.md

```diff
@@ -0,0 +1,121 @@
+- NDAO will be forged in two ways: realtime forging and staking with forged NDAO
+Realtime forging --- forge NDAOs with ETHs. People who get NDAOs can apply them in various applications.Staking with forged NDAO --- For example, in ETH 2.0 staking, a validator needs to stake 32 ETHs. By using staking with forged NDAO a validator can stake NDAOs instead of ETHs by sending ETHs to the system in exchange for NDAOs as stake. When the validator exits he/she will get back his/her staked NDAOs.
+- The RICP's algorithm runs this way:
+It is initialized with ETH's circulating supply.
+When a trader sends an amount of ETHs to the system, the ETH's circulating supply will be deducted by this amount and the total supply of ETH in the trading pair will be deducted by this amount. We can think of as these ETHs are burned.
+For instance if ETH's total supply is 107,477,373 , a trader sends 100 ETHs to this system in exchange for NDAOs, then the total supply of ETH in the algorithm will be 107,477,273 which will be less than the previous supply by 100, and the ETH's price will rise based on the RICP algorithm.
+- The total supply of ETH in the system and the total supply of NDAO are calculated once on a daily basis.
+- In this system ETH is priced in NDAO.
+
+Note: right after the foring contracts start, NDAO's pegging relationship with USD will be revoked.
+## Improvement to Economic Incentives
+Since the first exchange of Bitcoin came into existence, it has been widely acknowledged that crypto prices are highly volatile. This applies to any crypto including ETH. A huge number of ETH miners appeared but very few survive and beome big. Most of them were eventually kicked out of the market due to unstable income caused by ETH price fluctuations. Therefore it is improper to reward ETH miners with ETHs.
+
+We propose to reward ETH miners with NDAO.
+
+-Based on the RICP algorithm, no matter how ETH's price changes, NDAO's price is always as stable as 1. It is an absolutely stable token.  
+As presented in the previous sections we can use the RICP algorithm to calculate how many NDAOs should be rewarded to a validator by converting the ETHs he/she should receive to NDAOs and send them NDAOs instead of ETHs by smart contracts.
+- Validators retrieve NDAOs (as his reward ) from the ETH Forging smart contracts.
+- Our proposal can reward validators on a daily basis. This is more attractive to validators who can only get their rewards after Phase 1 in the current mechanism.
+- ETH's inflation rate will become lower.
+- NDAO will be in great demand. Finally validators will need a huge amount of NDAOs for staking. And NDAO will have far more applications than DAI. Interested users can give it a shot on www.DAism.io/exchange (Ropsten Test Network).
+
+## Backwards Compatibility
+[EIP-2569](https://github.com/DAism2019/EIPs/blob/master/EIPS/eip-2569.md).
+
+[EIP-2569 Pull](https://github.com/ethereum/EIPs/pull/2569).
+
+## Conclusion
```

Comment:
Not a template section.  Keep in mind that EIPs are standards, not papers.  This should be structured as a specification for how to do something, not a discussion of an idea.  If you just want to talk about your idea, I recommend the Ethereum research forums or a GitHub issue here.

Reply:
OK, I'll follow a template to modify the EIP.